src/racing/memory.c

```diff
@@ -30,6 +24,48 @@
 s32 sGfxSeekPosition;
 s32 sPackedSeekPosition;
 
+/* Opcodes packés (parité avec tools/displaylist_packer.c) */
+enum PackedOp {
+    PG_LIGHTS_0               = 0x00, /* 0..0x14 mappés sur unpack_lights */
+    /* Presets de combine renommés pour refléter les macros G_CC_* */
```

Comment:
Please translate the above into English for us monolingual primitives.

Reply:
oh use IA who put a lot of thing in franch but I don't think we still need it